src/System.CommandLine.Tests/TrimmingTests.cs

```diff
@@ -24,15 +25,29 @@ public TrimmingTests(ITestOutputHelper output)
         _systemCommandLineDllPath = typeof(Command).Assembly.Location;
     }
 
-    [ReleaseBuildOnlyFact]
-    public void App_referencing_system_commandline_can_be_trimmed()
+    [ReleaseBuildOnlyTheory]
+    [InlineData("")]
+    [InlineData("-p:PublishSingleFile=true")]
+    public void App_referencing_system_commandline_can_be_trimmed(string additionalArgs)
     {
         var stdOut = new StringBuilder();
         var stdErr = new StringBuilder();
+
+        var workingDirectory = Path.Combine(Directory.GetCurrentDirectory(), "TrimmingTestApp");
         
+        Process.RunToCompletion(
+            DotnetMuxer.Path.FullName,
+            "clean -c Release -r win-x64",
```

Comment:
Does `clean` need a `-c` or a `-r`? Doesn't it just need to be `dotnet clean` and it does it all?